examples/python/python2/__main__.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python2
+
+import argparse
+import json
+import sys
+import httplib
+import pprint
```

Comment:
nit: It may be good to sort the `import`'s alphabetically.  Not necessary though.